core/src/main/scala/kafka/utils/ZkUtils.scala

```diff
@@ -50,6 +57,37 @@ object ZkUtils extends Logging {
   val EntityConfigPath = "/config"
   val EntityConfigChangesPath = "/config/changes"
 
+  /** true if java.security.auth.login.config is set to some jaas file which has "Client" entry. **/
+
+  val isSecure: Boolean = {
+    val loginConfigurationFile: String = System.getProperty("java.security.auth.login.config")
```

Comment:
We could set the config in server.properties and set the system property during server startup but I am not sure what are we exactly providing by moving away from the defacto way of using jaas. Once we go that route we will have to deal with question of what to do if jaas property is passed through command line and server.properties also has the property set to a different jaas file. I feel doing it through server.properties will just cause more confusion and add un needed complexity.

 ZkClient also assumes that the jaas file is configured using system property, it looks for the Client section and login using that section. If we don't set the system property at all we will have to duplicate that code in kafka. 

In any case if we want to provide a server.properties and set the system property in kafka code, that should all happen outside of this code somewhere in KafkaServer. I think it is ok to assume here that the system property will be set by the time this code is invoked but I need to confirm when does Scala Object instances are initialized.